### PR TITLE
Update mite extension

### PR DESCRIPTION
**Pull request by ramsaysewell**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by ramsaysewell** [comment]
Fixes part of this issue #12931

**Comment by paoloricciuti** [comment]
Hey @ramsaysewell thanks for this...i think this is missing something but i might be wrong (i can't actually remember how the extension handle this).

Right now i'm in a self imposed stop from OSS because i kinda burned out but i will likely look at this next week 😄

File: extensions/mite/src/time-entry.tsx [hunk 1]
```diff
@@ -16,6 +16,11 @@ export default function Command() {
       });
       if (res.ok) {
         showToast({ title: "Entry created", style: Toast.Style.Success });
+        setTimeout(() => {
+          popToRoot({
+            clearSearchBar: false,
+          });
+        }, 1000);
```
Comment (pernielsentikaer) [inline extensions/mite/src/time-entry.tsx:23]: Is it necessary to use `setTimeout` here?

Comment (ramsaysewell) [reply extensions/mite/src/time-entry.tsx]: hey @pernielsentikaer, I've removed this now in the latest commit. This was me following the UX flow found in `setup-fast-time-entry`. I've also removed the setTimeout from this too as I believe it's not necessary either.

Comment (paoloricciuti) [reply extensions/mite/src/time-entry.tsx]: I've added the `setTimeout` to allow the user to see the form being submitted...i didn't like to be immediately moved outside without actually have any confirmation that the form was submitted.

Comment (pernielsentikaer) [reply extensions/mite/src/time-entry.tsx]: I think it makes sense to remove so we're using the same flow as the rest of Raycast 🙂

**Comment by mil3na** [comment]
Hi! Is this ready for another review?

**Comment by paoloricciuti** [comment]
> Hi! Is this ready for another review?

Sorry completely forgot about that...good for me, we can merge!

**Review by mil3na — APPROVED** [review]

**Pull request merged**